agent/github_tools.py

```
@@ -487,7 +503,7 @@ async def init_workspace(owner: str, repo: str, request: Request, body: Workspac
               repo_workspace=f"https://github.com/{owner}/{repo}")
         return result
     except Exception as e:
-        raise HTTPException(status_code=500, detail=str(e))
+        raise HTTPException(status_code=500, detail="Internal server error")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)